src/SFA.DAS.Tools.Support.Web/Views/Approvals/SearchApprenticeships.cshtml

```diff
@@ -0,0 +1,244 @@
+                </div>
+                <div class="govuk-form-group">
+                    <label asp-for="CourseName" class="govuk-label">
+                        Course Name
+                    </label>
+                    <span id="course-name-hint" class="govuk-hint">
+                        The name of the course for the apprenticeship.
+                    </span>
+                    <input asp-for="CourseName" id="courseName" class="govuk-input govuk-input--width-20" />
+                </div>
+                <div class="govuk-form-group">
+                    <label asp-for="SearchTerm" class="govuk-label">
+                        Search Term
+                    </label>
+                    <span id="search-term-hint" class="govuk-hint">
+                        Generic search term to use finding an apprenticeship.
+                    </span>
+                    <input asp-for="SearchTerm" id="searchTerm" class="govuk-input govuk-input--width-20" />
+                </div>
+                <div class="govuk-form-group">
+                    <label asp-for="StartDate" class="govuk-label">
+                        Start Date
+                    </label>
+                    <span id="start-date-hint" class="govuk-hint">
+                        The start date of the apprenticeship.
+                    </span>
+                    <input asp-for="StartDate" id="startDate" class="govuk-input govuk-input--width-20" />
+                </div>
+                <div class="govuk-form-group">
+                    <label asp-for="EndDate" class="govuk-label">
+                        End Date
+                    </label>
+                    <span id="end-date-hint" class="govuk-hint">
+                        The end date of the apprenticeship.
+                    </span>
+                    <input asp-for="EndDate" id="endDate" class="govuk-input govuk-input--width-20" />
+                </div>
+                <div class="govuk-form-group">
+                    <label asp-for="SelectedStatus" class="govuk-label">
+                        Apprenticeship Status
+                    </label>
+                    <span id="end-date-hint" class="govuk-hint">
+                        The current status of the apprenticeship.
+                    </span>
+                    @Html.DropDownListFor(model => model.SelectedStatus, new SelectList(Model.Statuses, "Value", "Text"), htmlAttributes: new { @class = "govuk-input govuk-input--width-20" })
+                </div>
+                <div class="govuk-form-group">
+                    <button class="govuk-button" data-module="govuk-button" id="submitSearchFormButton">
+                        Submit
+                    </button>
+                </div>
+            </form>
+        </div>
+    </div>
+</div>
+<div id="searchResults" style="display:none">
+    <table id="apprenticeshipResultsTable"
+           data-toggle="table"
```

Comment:
add in data-search so the data in the table can be searched

Reply:
Added